wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,261 @@
+- [Summer](https://osu.ppy.sh/forum/t/303353) by [Raidexx](https://osu.ppy.sh/u/2244606)![][o!s]![][o!c]![][o!m]
+- [Sunset Horizons](https://osu.ppy.sh/forum/t/646047) by [Flore](https://osu.ppy.sh/u/3344355)![][o!s]
+
+## T
+
+- [taikomania IIDX 13 DonderfuL](https://osu.ppy.sh/forum/t/151549) by [crystalsuicune](https://osu.ppy.sh/u/9974)![][o!t]
+- [Tales of Zestiria ♥](https://osu.ppy.sh/forum/t/534227) by [ 5HARO](https://osu.ppy.sh/u/8738003)![][o!s]![][o!c]![][o!m]
+- [TECHNOCANDY](https://osu.ppy.sh/forum/t/321844) by [Garin](https://osu.ppy.sh/u/2130664)![][o!s]
+- [TECHNOGALAXY](https://osu.ppy.sh/forum/t/356379) by [Garin](https://osu.ppy.sh/u/2130664)![][o!s]
+- [Tesseract](https://osu.ppy.sh/forum/t/540478) by [Duoquote](https://osu.ppy.sh/u/4920617)![][o!s]![][o!m]
+- [Tewi Inaba](https://osu.ppy.sh/forum/t/228705) by [S A V E R Y](https://osu.ppy.sh/u/3991274)![][o!s]![][o!c]
+- [The Emoji Skin](https://osu.ppy.sh/forum/t/654024) by [h3oCharles](https://osu.ppy.sh/u/8712289)![][o!s]![][o!t]![][o!c]![][o!m]
+- [The Furry Skin!](https://osu.ppy.sh/forum/t/221594) by [TaitoTheHusky](https://osu.ppy.sh/u/2866135)![][o!s]
+- [To Love Ru](https://osu.ppy.sh/forum/t/338041) by [Darth_Mula](https://osu.ppy.sh/u/4963834)![][o!s]
+- [Twilight Sparkle](https://osu.ppy.sh/forum/t/123845) by [ztrot](https://osu.ppy.sh/u/6347)![][o!s]
+
+## U
+
+- [Unbreakable Machine-Doll](https://osu.ppy.sh/forum/t/162662) by [Kurokami](https://osu.ppy.sh/u/260933)![][o!s]![][o!t]![][o!c]![][o!m]
+- [Undertale Skin](https://osu.ppy.sh/forum/t/590196) by [madalyne11](https://osu.ppy.sh/u/6877973)![][o!s]![][o!c]
+
+## V
+
+- [Vicinity](https://osu.ppy.sh/forum/t/121785) by [DJNightmare](https://osu.ppy.sh/u/70909)![][o!m]
+- [Vocaloid Project Skin](https://osu.ppy.sh/forum/t/82739) by [NoClue](https://osu.ppy.sh/u/207167)![][o!s]![][o!t]![][o!c]![][o!m]
+- [VOID II DESTRUCTION](https://osu.ppy.sh/forum/t/588628) by [Ben_5917](https://osu.ppy.sh/u/6026593)![][o!s]![][o!t]![][o!c]![][o!m]
+
+## W
+
+- [Winter..](https://osu.ppy.sh/forum/t/249855) by [Garin](https://osu.ppy.sh/u/2130664)![][o!s]![][o!m]
+- [WonderVision3](https://osu.ppy.sh/forum/t/69448) by [AussieEvil](https://osu.ppy.sh/u/407003)![][o!s]![][o!t]![][o!m]
+
+## X
+
+- [Xenon](https://osu.ppy.sh/forum/t/217557) by [Chikilin00 ](https://osu.ppy.sh/u/3655922)![][o!s]![][o!m]
+- [Xi-Style](https://osu.ppy.sh/forum/t/4833) by [Xiao](https://osu.ppy.sh/u/17894)![][o!s]
+- [Xi-Style G2](https://osu.ppy.sh/forum/t/59787) by [Xiao](https://osu.ppy.sh/u/17894)![][o!s]![][o!t]![][o!m]
+
+## Y
+
+- [Yet Another Miku Skin](https://osu.ppy.sh/forum/t/450498) by [rkgk](https://osu.ppy.sh/u/7620476)![][o!s]
+- [YUGEN](https://osu.ppy.sh/forum/t/365036) by [Garin](https://osu.ppy.sh/u/2130664)![][o!s]![][o!t]![][o!c]![][o!m]
+- [Yuzuki Yukari](https://osu.ppy.sh/forum/t/342349) by [Elzapatoverde](https://osu.ppy.sh/u/3717733)![][o!s]![][o!c]
+
+## #
```

Comment:
I guess this actually doesn't work, change to:
```
## \#
```
(insert a `\` right before the third `#`)

Reply:
or change this to `## Other` or `## Others`